pkgs/by-name/ne/nezha-agent/package.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  nezha-agent,
+  testers,
+}:
+buildGoModule rec {
+  pname = "nezha-agent";
+  version = "0.16.3";
+
+  src = fetchFromGitHub {
+    owner = "nezhahq";
+    repo = "agent";
+    rev = "v${version}";
+    hash = "sha256-DeZzR25KWxA70nS0mICaOb0TeBhnurm3FWLNS/EjwZA=";
+  };
+
+  vendorHash = "sha256-b8jICFsQaSetBzhTiq8zBX/LYuhQNly5kj45ZV6NDiY=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X main.version=${version}"
+  ];
+
+  preCheck = ''
+    rm ./pkg/monitor/myip_test.go
```

Comment:
I assume the test fails but a comment would be good to document this.

Reply:
This test makes a request to get geoip data to test whether it works correctly. But we build it in the sandbox without network.